code/modules/antagonists/bloodsucker/powers/v_distress.dm

```diff
@@ -0,0 +1,13 @@
+
+
+
+/*
+/datum/action/bloodsucker/vassal/distress
```

Comment:
This is quite the anathema of all the commented stuff on this PR.
I'm not going to ask you to remove those comments, most of them make sense and it'd be too pretentious to expect newly introduced features to be lavishly complete anyway.
But please move this commented wip into another file.

Reply:
Ill just remove it, it was previously just a empty action